src/libsystemd/sd-journal/compress.h

```diff
@@ -5,6 +5,11 @@
 
 #include "journal-def.h"
 
+#define OBJECT_COMPRESSED_NONE 0
+#define OBJECT_COMPRESSED_ZSTD 1
+#define OBJECT_COMPRESSED_LZ4 2
+#define OBJECT_COMPRESSED_XZ 3
```

Comment:
Is it not necessary to be defined in config.h? Otherwise, config.h is not self-contained.

At least, please align them.

Reply:
Builds were fine - but I've moved them now